Sming/SmingCore/SystemClock.h

```diff
@@ -63,11 +65,20 @@ class SystemClockClass
      *  @todo   Why does this need to be set to 2 for UK during winter?
      *  @note   Supports whole hour and fraction of hour offsets from -12 hours to +12 hours
      */
-	bool setTimeZone(double localTimezoneOffset);
+	bool setTimeZoneOffset(int tzOffsetSecs);
```

Comment:
Rename `tzOffsetSecs` to `seconds`. The methods name specifies that this is a timezone offset.